cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -1021,6 +1021,90 @@ struct group_key_func {
   }
 };
 
+// merges all the tdigests within each group. returns a table containing 2 columns:
+// the sorted means and weights.
+template <typename GroupOffsetIter>
+std::unique_ptr<table> generate_merged_centroids(tdigest_column_view const& tdv,
```

Comment:
Indeed we don't need to return table/columns -- I don't see we store any bitmask and null count in them. Thus, it is cleaner to return `device_uvector`.